Makefile

```diff
@@ -1,17 +1,28 @@
-PREFIX=/usr/
-DESTDIR=/
-BINARY_NAME=apx
+PREFIX := /usr
+DESTDIR := /
+BINARY_NAME := apx
+
+GO := go
+SYSTEM_DISTROBOX :=
 
 all: build
 
-build:
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -tags netgo -ldflags '-w -extldflags "-static"' -o ${BINARY_NAME}
+build: ${BINARY_NAME}
+
+${BINARY_NAME}:
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 ${GO} build -a -tags netgo -ldflags '-w -extldflags "-static"' -o $@
 
+ifdef SYSTEM_DISTROBOX
```

Comment:
using a system distrobox would still require the apx config to be changed accordingly 
but I'm not sure if we even want to support using system distrobox installations

Reply:
Would there be maintenance burden on your side? I mean, it's the user's responsibility if things break.
As for the config issue, perhaps there can be two variables: a configuration option to install Distrobox (true by default), and the path to Distrobox

Reply:
yes, the maintenance comes with people using potentially unsupported/old versions of distrobox that don't work with apx

Reply:
Fair enough